includes/Core/Authentication/Clients/Google_Site_Kit_Client.php

```diff
@@ -243,4 +244,23 @@ protected function fetchAuthToken( OAuth2 $auth, callable $http_handler = null )
 	protected function handleAuthTokenErrorResponse( $error, array $data ) {
 		throw new Google_OAuth_Exception( $error );
 	}
+
+	/**
+	 * Executes deferred HTTP requests.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param RequestInterface $request Request object to execute.
+	 * @param string           $expected_class Expected class to return.
+	 * @return object An object of the type of the expected class or Psr\Http\Message\ResponseInterface.
+	 */
+	public function execute( RequestInterface $request, $expected_class = null ) {
```

Comment:
Would you please move this to be grouped with the other public methods?

Reply:
Yep, updated.